precompile/wasm/IWasm.sol

```diff
@@ -10,6 +10,54 @@ IWasm constant WASM_CONTRACT = IWasm(
 );
 
 interface IWasm {
+    // Events
+    /**
+     * @dev InstantiateContract defines an event emitted when a wasm contract is successfully
+     * instantiated via instantiateContract or instantiateContract2
+     * @param sender the address of the sender
+     * @param admin the address of the contract admin
+     * @param contractAddress the address of the instantiated contract
+     * @param codeId the id of contract
+     * @param label the label of contract
+     */
+    event InstantiateContract(
+        address indexed sender,
+        address indexed admin,
+        address indexed contractAddress,
+        uint256 codeId,
+        string label
```

Comment:
I think sender and codeID are meaningful as indexed data, and it seems like there's some event in leaving needs contractAddress, msgs, admin and funds.

Reply:
[49cb90b](https://github.com/xpladev/xpla/pull/176/commits/49cb90bb1dbd205e6789a2a59f7024cd06856bd5)